js/core/game.js

```diff
@@ -520,25 +520,11 @@ define(["dao", "globals", "ui", "core/freeAgents", "core/finances", "core/gameSi
                     p.ovr = rating.ovr;
 
                     // These use the same formulas as the skill definitions in player.skills!
-                    p.compositeRating.pace = makeComposite(rating, ['spd', 'jmp', 'dnk', 'tp', 'stl', 'drb', 'pss']);
-                    p.compositeRating.usage = Math.pow(makeComposite(rating, ['ins', 'dnk', 'fg', 'tp', 'spd', 'drb'], [1.5, 1, 1, 1, 0.15, 0.15]), 1.9);
```

Comment:
`usage` has `Math.pow` wrapped around the result. You should have a line that does this after your for loop.